<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>750</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>background2.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ar_guide_start">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>560</y>
     <width>440</width>
     <height>75</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>ar_guide_start.png</normaloff>ar_guide_start.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>1000</width>
     <height>98</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="other_route">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>560</y>
     <width>415</width>
     <height>75</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>other_route.png</normaloff>other_route.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>500</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>560</y>
     <width>345</width>
     <height>74</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>reset.png</normaloff>reset.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>460</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_show_camera">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>411</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="open_camera">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>300</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>On</string>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="projector_on">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>560</y>
     <width>490</width>
     <height>75</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>projector_on.PNG</normaloff>projector_on.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>500</width>
     <height>98</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
